modules/postfix.lua

```diff
@@ -0,0 +1,500 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+local ip = require 'ip_address'
+local l = require 'lpeg'
+l.locale(l)
+
+local M = {}
+-- setfenv(1, M) -- Remove external access to contain everything in the module
```

Comment:
You have to add anything from the global namespace to your local namespace before the `setfenv` call. So adding `local rawset = rawset` around line 8 should do the trick. See similar usage in the [syslog module](https://github.com/mozilla-services/lua_sandbox/blob/dev/modules/syslog.lua#L12).